sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -436,6 +436,19 @@ def create_job_description(self, job):
     # TODO(silviuc): Remove the debug logging eventually.
     logging.info('JOB: %s', job)
 
+  @retry.with_exponential_backoff()  # Using retry defaults from utils/retry.py
```

Comment:
Removed the comment.
You're right that the default would not be very good here. I set it to 3 retries after 3, 6, 12 seconds.